examples/js_callback/src/bindings.rs

```diff
@@ -1,11 +1,24 @@
 use wasm_bindgen::prelude::*;
 
-// wasm-bindgen will automatically take care of including this script
-#[wasm_bindgen(module = "/src/get-payload-script.js")]
+#[wasm_bindgen]
 extern "C" {
-    #[wasm_bindgen(js_name = "getPayload")]
-    pub fn get_payload() -> String;
+    // this should be in js-sys but is not. see https://github.com/rustwasm/wasm-bindgen/issues/2865
+    pub fn import(s: &str) -> js_sys::Promise;
 
-    #[wasm_bindgen(js_name = "getPayloadLater")]
-    pub fn get_payload_later(payload_callback: JsValue);
+    pub type Window;
+
+    #[wasm_bindgen(method, getter, js_name = "wasmBindgenSnippetsPath")]
+    pub fn wasm_bindgen_snippets_path(this: &Window) -> String;
+}
+
+#[wasm_bindgen(module = "/js/imp.js")]
+extern "C" {
+    #[wasm_bindgen]
+    pub fn hello() -> String;
+}
+
+#[wasm_bindgen(module = "/js/unimp.js")]
+extern "C" {
+    #[wasm_bindgen]
+    pub fn bye() -> String;
```

Comment:
This is not actually used anywhere, is it? Thank god that the dead code elimination actually removes the import from the `index-*.js` glue but preserves the snippet in the output dir. I think this `wasm_bindgen` block should contain only a private import, something like

```rust
#[wasm_bindgen(module = "/js/unimp.js")]
extern "C" {
    #[wasm_bindgen]
    fn __this_is_a_fake_import_that_exists_to_get_the_snippet() -> String;
}
```

Reply:
Yes, it's used anywhere. I made the function pub in testing and never removed it. That's something to be fixed

Reply:
Note that straight up removing *all* functions from the block (at least in my test) will not emit the snippet, a fake import is required and I couldn't figure out if there's a workaround.

Reply:
Yep, I added a fake import